### ISSUE TITLE
possible bug in TradeShips.lua file?

### ISSUE THREAD

**Issue by impaktor** [pioneerspacesim/pioneer#2625]
I've been looking in data/modules/TradeShips.lua

Or more specifically line 341-359 this is looking like a bug to me, that table.insert is only used in two of the ifs:

```
local prices = Game.system:GetCommodityBasePriceAlterations()
local import_score, export_score = 0, 0
imports, exports = {}, {}
for k,v in pairs(prices) do
    if k ~= 'RUBBISH' and k ~= 'RADIOACTIVES' and Game.system:IsCommodityLegal(k) then
        -- values from SystemInfoView::UpdateEconomyTab
        if      v > 10  then
            import_score = import_score + 2
        elseif  v > 2   then
            import_score = import_score + 1
            table.insert(imports, k)
        elseif  v < -10 then
            export_score = export_score + 2
        elseif  v < -2  then
            export_score = export_score + 1
            table.insert(exports, k)
        end
    end
end
```

then there's and 878-887, which I don't understand yet, when comparing with the code above, but will at some point.

```
local prices = Game.system:GetCommodityBasePriceAlterations()
for k,v in pairs(prices) do
    if k ~= 'RUBBISH' and k ~= 'RADIOACTIVES' and Game.system:IsCommodityLegal(k) then
        if v > 2 then
            table.insert(imports, k)
        elseif v < -2 then
            table.insert(exports, k)
        end
    end
end
```

**Comment by joonicks** [comment]
fixed by #4268

**Comment by richardpl** [comment]
Is this still relevant?
